src/main/java/ch/wisv/areafiftylan/dto/EventDTO.java

```diff
@@ -0,0 +1,28 @@
+package ch.wisv.areafiftylan.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.validator.constraints.NotEmpty;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * Created by Sille Kamoen on 27-3-16.
+ */
+public class EventDTO {
+
+    @Getter
+    @Setter
+    @NotEmpty
+    String name = "";
+
```

Comment:
This endpoint is not for the website, that's what the `/web` endpoints are for. There is some overlap in functionality, certainly when the intranet comes into play as well. This will probably move to the intranet alltogether for next year, with a dedicated `web` section that can be imported into the intranet. 2017 stuff.